test/dom-parser.test.js

```diff
@@ -330,7 +331,20 @@ describe('DOMParser', () => {
 			const source = `<root>${'A'.repeat(50000)}\u2029${'A'.repeat(50000)}\u0085${'A'.repeat(50000)}\u2028${'A'.repeat(50000)}\u2029</root>`;
 			const doc = parser.parseFromString(source, MIME_TYPE.XML_TEXT);
 			expect(new XMLSerializer().serializeToString(doc)).toEqual(source.replace(/[\u0085\u2028\u2029]/g, '\n'));
-		}, 500);
+		});
+		test('should be able to open documents with alternative whitespace without creating a bottleneck and replacing them with \\n', () => {
+			// issue: https://github.com/xmldom/xmldom/issues/838
+			const start = performance.now();
+			const onError = jest.fn();
+			const normalizeLineEndings = jest.fn((source) => source);
+			const { parser } = getTestParser({ onError, normalizeLineEndings });
+			const source = `<root>${'A'.repeat(15000)}\u2029${'A'.repeat(15000)}\u0085${'A'.repeat(15000)}\u2028${'A'.repeat(15000)}\u2029</root>`;
```

Comment:
I think it would make more sense to have a `\n` as the first character, since the test currently has `\u2029` twice.

Solution:
```js
const source = `<root>${'A'.repeat(15000)}\n${'A'.repeat(15000)}\u0085${'A'.repeat(15000)}\u2028${'A'.repeat(15000)}\u2029</root>`;
```

Reply:
I don't think it's such big of a difference, but hey, I applied your suggestion.